src/components/Home.js

```diff
@@ -25,6 +30,14 @@ Home.addEventListeners = () => {
       window.renderApp()
     }
   })
+
+  var deleteButton = document.querySelectorAll('.delete-task')
+  deleteButton.forEach(element => {
+    element.addEventListener('click', event => {
+      tasks.splice(event.toElement.id.substring(7), 1);
```

Comment:
`toElement` is not standard and will not work on Firefox.
Please use `target` instead.

Reply:
yes, i agree ;)